src/static/riot/profiles/profile_detail.tag

```diff
@@ -3,60 +3,86 @@
-                <div if="{selected_user.photo}"><img id="avatar" class="ui centered rounded image" src="{selected_user.photo}"></div>
-
+                <div if="{!selected_user.photo}"><img id="avatar" class="ui centered small rounded image" src="http://via.placeholder.com/150x150"></div>
+                <div if="{selected_user.photo}"><img id="avatar" class="ui centered small rounded image" src="{selected_user.photo}"></div>
+
+<!--                &lt;!&ndash; Competition Divider &ndash;&gt;-->
+<!--                <div class="ui horizontal divider">Competitions</div>-->
+
+<!--                &lt;!&ndash; Competition Cards &ndash;&gt;-->
+<!--                <div class="ui fluid card">-->
+<!--                    <div class="comp_card center aligned image">-->
+<!--                        <div  class="comp_header center aligned header content">-->
+<!--                            <div class="comp_name">Competition Name</div>-->
+<!--                            <img class="ui centered circular image"-->
+<!--                                 src="http://via.placeholder.com/50x50">-->
+<!--                        </div>-->
+<!--                    </div>-->
+<!--                    <div class="content">-->
+<!--                        <div class="description">-->
+<!--                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci asperiores atque-->
+<!--                                distinct...</p>-->
+<!--                        </div>-->
+<!--                    </div>-->
+<!--                    <div class="right aligned extra content">-->
+<!--                        <a class="status">-->
+<!--                            <i class="exchange icon"></i>-->
+<!--                            In Progress-->
+<!--                        </a>-->
+<!--                    </div>-->
+<!--                </div>-->
+<!--                <div class="ui fluid card">-->
+<!--                    <div class="comp_card center aligned image">-->
+<!--                        <div  class="comp_header center aligned header content">-->
+<!--                            <div class="comp_name">Competition Name</div>-->
+<!--                            <img class="ui centered circular image"-->
+<!--                                 src="http://via.placeholder.com/50x50">-->
+<!--                        </div>-->
+<!--                    </div>-->
+<!--                    <div class="content">-->
+<!--                        <div class="description">-->
+<!--                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci asperiores atque-->
+<!--                                distinct...</p>-->
+<!--                        </div>-->
+<!--                    </div>-->
+<!--                    <div class="right aligned extra content">-->
+<!--                        <a class="status">-->
+<!--                            <i class="exchange icon"></i>-->
```

Comment:
need?

Reply:
This was an example of how competitions could look like on profiles. I didn't implement this on the working page but figured we could leave the code in case someone wanted to refer to it.